internal/cli/cmd/clustermesh.go

```diff
@@ -163,6 +163,7 @@ func newCmdClusterMeshStatus() *cobra.Command {
 	cmd.Flags().BoolVar(&params.Wait, "wait", false, "Wait until status is successful")
 	cmd.Flags().DurationVar(&params.WaitDuration, "wait-duration", 15*time.Minute, "Maximum time to wait")
 	cmd.Flags().BoolVar(&params.SkipServiceCheck, "skip-service-check", false, "Do not require service IP of remote cluster to be available")
+	cmd.Flags().StringVarP(&params.Output, "output", "o", "summary", "Output format. One of: json")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&params.Output, "output", "o", "summary", "Output format. One of: json, summary")
```

Reply:
I just notice the previous discussion about this: https://github.com/cilium/cilium-cli/pull/958#discussion_r962268710

I don't think the default value is added to the list automatically as Tam suggested. There is additional text added to say what the default value is, though:

```
  -o, --output string            Output format. One of: json (default "summary")
```

But I think that `summary` should still be included in the "One of" list, as we do for similar flags in the per-node `cilium` binary.